package-lock.json

```diff
@@ -14765,8 +14765,9 @@
     },
     "node_modules/pluralize": {
       "version": "8.0.0",
+      "resolved": "https://registry.npmjs.org/pluralize/-/pluralize-8.0.0.tgz",
+      "integrity": "sha512-Nc3IT5yHzflTfbjgqWcCPpo7DaKy4FnpB0l/zCAW0Tc7jxAiuqSxHasntB3D7887LSrA93kDJ9IXovxJYxyLCA==",
```

Comment:
This looks like a leftover as we're not using the pluralize package.

Reply:
We don't have any package json change. Discarding this file altogether.